src/Features/Core/Portable/ConvertAnonymousTypeToTuple/AbstractConvertAnonymousTypeToTupleDiagnosticAnalyzer.cs

```diff
@@ -24,7 +24,7 @@ protected AbstractConvertAnonymousTypeToTupleDiagnosticAnalyzer()
         protected abstract int GetInitializerCount(TAnonymousObjectCreationExpressionSyntax anonymousType);
 
         public override DiagnosticAnalyzerCategory GetAnalyzerCategory()
-            => DiagnosticAnalyzerCategory.SyntaxAnalysis;
```

Comment:
Can we also rename `SyntaxAnalysis` to `SyntaxTreeAnalysis`?

Reply:
Can i do that in a separate PR? :)

Reply:
(I'm working on that now.  But i'd like them to be separate.

Reply:
Certainly.